src/ansiblelint/schemas/__store__.json

```diff
@@ -36,8 +36,8 @@
     "url": "https://raw.githubusercontent.com/ansible/ansible-lint/main/src/ansiblelint/schemas/molecule.json"
   },
   "pattern": {
-    "etag": "9f427c883b1e069c1d9cb938f5c2d6d9d6412250ba781880cce3ae7b523bf035",
-    "url": "https://raw.githubusercontent.com/ansible/ansible-lint/main/src/ansiblelint/schemas/pattern.json"
+    "etag": "3c91293eaa0bd9b84eee2c3f34c2619194acc6f9073164332c17b9cc384b350d",
+    "url": "https://raw.githubusercontent.com/ansible/pattern-service/refs/heads/main/specifications/pattern-schema/pattern-schema-latest.json"
```

Comment:
why not:

https://raw.githubusercontent.com/ansible/pattern-service/main/specifications/pattern-schema/pattern-schema-latest.json